src/ImageLoaders/IHex32/IHex32Exception.cs

```diff
@@ -0,0 +1,106 @@
+﻿#region License
+/* 
+ * Copyright (C) 2017-2018 Christian Hostelet.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; see the file COPYING.  If not, write to
+ * the Free Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ */
+#endregion
+
+using System;
+using System.Runtime.Serialization;
+
+namespace Reko.ImageLoaders.IHex32
+{
+    public class IHEX32Exception : Exception
```

Comment:
This is not interface. So why it is started with `I` prefix?
Does `I` mean `Intel`? In this case the name should be `IntelHEX32Exception`